packages/plugin-apex/src/commands/force/apex/test/run.ts

```diff
@@ -176,25 +168,16 @@ export default class Run extends SfdxCommand {
         this.flags.classnames,
         this.flags.suitenames
       );
+
       payload.skipCodeCoverage = this.flags.codecoverage ? false : true;
       const reporter = undefined;
-      if (this.flags.resultformat !== undefined) {
-        result = await testService.runTestAsynchronous(
-          payload,
-          this.flags.codecoverage,
-          false,
-          reporter,
-          this.cancellationTokenSource.token
-        );
-      } else {
-        result = await testService.runTestAsynchronous(
-          payload,
-          this.flags.codecoverage,
-          true,
-          reporter,
-          this.cancellationTokenSource.token
-        );
-      }
+      result = await testService.runTestAsynchronous(
```

Comment:
Just want to confirm something here.  If synchronous (this.flags.synchronous = true) and testLevel != TestLevel.RunSpecifiedTests, then `runTestAsynchronous` should be called?

Reply:
Yes, it feels weird for sure. However the end API doesn't accept more than a few synchronous tests at a time, so if they don't tell us which tests they want, we can't process that synchronously. And apparently users complained when we gave them an error message telling them they couldn't 🤷🏻‍♀️

Reply:
ty